src/components/frontegg-context/types.ts

```diff
@@ -1,32 +1,35 @@
-import { IIORedisCacheOptions, IRedisCacheOptions } from '../../cache/types';
+import { IIORedisOptions, IRedisOptions } from '../cache/managers';
 
 export interface IFronteggContext {
   FRONTEGG_CLIENT_ID: string;
   FRONTEGG_API_KEY: string;
 }
 
 export interface IFronteggOptions {
+  cache: IFronteggCacheOptions;
   accessTokensOptions?: IAccessTokensOptions;
 }
 
 export interface IAccessTokensOptions {
-  cache: IAccessTokensLocalCache | IAccessTokensIORedisCache | IAccessTokensRedisCache;
+  cache: IFronteggCacheOptions;
 }
 
-export interface IAccessTokensCache {
+export interface IAccessTokensCacheOptions {
   type: 'ioredis' | 'local' | 'redis';
 }
 
-export interface IAccessTokensLocalCache extends IAccessTokensCache {
+export interface ILocalCacheOptions extends IAccessTokensCacheOptions {
```

Comment:
These types are breaking changes, so either keep it the same name as before, or export a type with the previous name that is equal to this interface